superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -79,32 +83,11 @@ interface ActivityProps {
   loadedCount: number;
 }
 
-const ActivityContainer = styled.div`
-  margin-left: ${({ theme }) => theme.gridUnit * 2}px;
-  margin-top: ${({ theme }) => theme.gridUnit * -4}px;
-  display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(31%, max-content));
-
-  grid-gap: ${({ theme }) => theme.gridUnit * 8}px;
-  justify-content: left;
-  padding: ${({ theme }) => theme.gridUnit * 6}px;
-  padding-top: ${({ theme }) => theme.gridUnit * 2}px;
-  .ant-card-meta-avatar {
-    margin-top: ${({ theme }) => theme.gridUnit * 3}px;
-    margin-left: ${({ theme }) => theme.gridUnit * 2}px;
-    color: ${({ theme }) => theme.colors.grayscale.base};
-  }
-  .ant-card-meta-title {
-    font-weight: ${({ theme }) => theme.typography.weights.bold};
-  }
-  ${mq[3]} {
-    grid-template-columns: repeat(auto-fit, minmax(31%, max-content));
-  }
-  ${mq[2]} {
-    grid-template-columns: repeat(auto-fit, minmax(42%, max-content));
-  }
-  ${mq[1]} {
-    grid-template-columns: repeat(auto-fit, minmax(80%, max-content));
+const Styles = styled.div`
+  .recentCards {
+    max-height: none;
+    grid-gap: ${({ theme }) =>
+      `${theme.gridUnit * 2.5}px ${theme.gridUnit * 5.5}px`};
```

Comment:
How's this look without the fractional gridUnits? Happy to hop on a call and play around with this. A lot of gutters in Superset are 16px, so I wonder if 2 and 4 wouldn't look good... hard to say without seeing. Maybe 2 and 5, or 3 and 6. But I try to avoid fractions.

Reply:
This is so we can keep the same grid-gap as the other tables. The other tables need a longer bottom gap so that they did not show inside of the container on line wrap.